src/codewhisperer/vue/root.vue

```diff
@@ -0,0 +1,223 @@
+                        fill="url(#paint0_linear_227_4962)"
+                    />
+                    <defs>
+                        <linearGradient
+                            id="paint0_linear_227_4962"
+                            x1="53.5957"
+                            y1="-883.34"
+                            x2="19.5336"
+                            y2="-883.327"
+                            gradientUnits="userSpaceOnUse"
+                        >
+                            <stop stop-color="#6EB1FF" />
+                            <stop offset="1" stop-color="#DA8AF7" />
+                        </linearGradient>
```

Comment:
I am curious how do we come up with all these magic numbers?

Reply:
Its a SVG file provided by Ray. Just used it here.